python/nav/ipdevpoll/shadows/__init__.py

```diff
@@ -299,7 +299,7 @@ def _post_events_version_changes(self):
         for alert_type, old_version in self.changed_versions.items():
             self.event.notify(
                 device=device,
-                subid=device.id,
+                netbox=device.get_preferred_related_object().netbox,
```

Comment:
Ooh, I have one more nitpick now... `get_preferred_related_object()` may actually return `None`, which would cause this to crash.

The job always runs in the context of a specific Netbox, so you already have the information about the related netbox. The idiomatic way of getting the `Netbox` shadow instance for the current job is through `containers.get(None, shadows.Netbox)` (but, of course, you need to pass the containers argument as well).